modules/post/windows/manage/persistence_exe.rb

```diff
@@ -34,10 +34,18 @@ def initialize(info = {})
     register_options(
       [
         OptEnum.new('STARTUP', [true, 'Startup type for the persistent payload.', 'USER', ['USER', 'SYSTEM', 'SERVICE']]),
-        OptPath.new('REXEPATH', [true, 'The remote executable to use.']),
+        OptPath.new('REXEPATH', [true, 'The remote executable to upload and execute.']),
         OptString.new('REXENAME', [true, 'The name to call exe on remote system', 'default.exe'])
       ], self.class
     )
+
+    register_advanced_options(
+      [
+        OptString.new('LEXEPATH', [false, 'The local exe path to run. Use temp directory as default. ']),
+        OptString.new('STARTUP_NAME',   [false, 'The name of service or registry. Random string as default.' ]),
+        OptString.new('SERVICE_DESC',   [false, 'The description of service. Random string as default.' ])
```

Comment:
The advanced datastore options should be camel style i.e. LocalExePath, StartupName, and ServiceDescription

Reply:
I have fixed it. 
Thanks for your advice.